lib/contentTypeParser.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
-const lru = require('tiny-lru')
+let lru = require('tiny-lru')
+lru = typeof lru === 'function' ? lru : lru.default
```

Comment:
I think this should have had a comment explaining itself. I certainly will not know what this is doing if I come across it in 6 months.